docs/UserGuide.md

```diff
@@ -308,158 +308,158 @@ the constraints of each parameter when used in a command.
-> * Different patients may have overlapping appointment timings, as long as overlapping appointments are not from the same patient 
+> * Different patients may have overlapping appointment timings, as long as those appointments are not from the same patient.
+
 
 {: .alert .alert-success}
 > :bulb: **Tip:**
 >
-> Input two new appointments to represent any appointments that span across date boundaries. 
-> * Input the first ending at `2359`: e.g. `addAppt Transfusion i/S1234567A @d/2024-10-23 @t/2200-2359`; and
-> * Input the next starting on the following date at `0000`: e.g. `addAppt Transfusion i/S1234567A @d/2024-10-24 @t/0000-0100`.
+> For appointments that span across date boundaries, add two appointments:
+> * End the first appointment at `2359` (e.g., `addAppt Transfusion i/S1234567A @d/2024-10-23 @t/2200-2359`).
+> * Start the next appointment at `0000` on the following date (e.g., `addAppt Transfusion i/S1234567A @d/2024-10-24 @t/0000-0100`).
 
 {: .alert .alert-success}
 > :bulb: **Tip:**
 >
-> Appointment end timings are **exclusive** - the next appointment can start immediately after the previous.
->
-> If the time period of _Appointment A_ is `1200-1300`:
-> * the time period of _Appointment B_ can be `1300-1400`.
-> * the time period of _Appointment B_ can be `1300-1400`.
-> * _Appointment B_'s time period **cannot** start anywhere from `1200` to `1259`.
-> * _Appointment B_'s time period **cannot** end anywhere from `1201` to `1300`.
+> Appointment end times are **exclusive**. For example:
+> * If _Appointment A_ is scheduled from `1200` to `1300`, _Appointment B_ can start at `1300`.
+> * _Appointment B_ cannot start between `1200` and `1259`, nor end between `1201` and `1300`.
 
 [Back to Table of Contents](#table-of-contents)
 
-#### Deleting Appointments : `delAppt`
+#### Deleting an Appointment : `delAppt`
 
-Allow doctors to delete an appointment. 
+Deletes a scheduled appointment from MediBase3.
 
-Format: `delAppt i/NRIC @d/APPOINTMENT_DATE @t/APPOINTMENT_TIME`
+**Format**: `delAppt i/NRIC @d/APPOINTMENT_DATE @t/APPOINTMENT_TIME`
 
 {: .alert .alert-info}
 > :information_source: **Note:**
-> 
-> * Unlike addAppt, there is no need for `APPOINTMENT_NAME` preamble.
-> * Deletes Appointment from the patient with the specified `NRIC` in MediBase3.
-> * You can delete an appointment from a patient even if they are not being currently displayed in the Patient List Panel but doing so will refresh the panel to display all patients after their appointment has been deleted.
-> * Refer to the [Parameter Details](#parameter-details) section for more information on the purpose and constraints of each parameter.
+>
+> * To delete an appointment, only `NRIC`, `APPOINTMENT_DATE`, and `APPOINTMENT_TIME` are required
+>   - **No need** for the `APPOINTMENT_NAME`.
+> * If the patient is not currently displayed in the Patient List Panel, the panel will refresh to show all patients after the appointment is deleted.
```

Comment:
![](https://private-user-images.githubusercontent.com/116947122/383838975-7572bf2d-54be-4d4d-8f34-77deca036815.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA5NjIyNTMsIm5iZiI6MTczMDk2MTk1MywicGF0aCI6Ii8xMTY5NDcxMjIvMzgzODM4OTc1LTc1NzJiZjJkLTU0YmUtNGQ0ZC04ZjM0LTc3ZGVjYTAzNjgxNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTA3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwN1QwNjQ1NTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iOTBjNDdjZTJlOWYyYzZhMGY4M2QxNmNiZWI2MDVkNGVlMTMwNzcxYWE3YTNkMDg2ZWFlNzc4YzNmNzAxNjZmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.5D-M860orv05Jtoh6oZhFPAV2-M3vitW31_A0Mi0Xhc)

I think can follow the msg format for consistency